proposals/lip-0067.md

```diff
@@ -199,7 +199,8 @@ There are no incompatibilities since the protocol is not changed.
 
 ## Reference Implementation
 
-TBD
+- [Add semgrep config and fix identified issues #8593](https://github.com/LiskHQ/lisk-sdk/pull/8593)
```

Comment:
Manu shared this link with me: https://github.com/LiskHQ/lisk-desktop/blob/development/src/modules/account/utils/encryptAccount.js. It encodes a keystore for a wallet account, so exactly what the LIP does.